walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -164,176 +121,141 @@ After the mobile app is registered, it is displayed on the console.
-
-DataSync framework will offer fully featured offline data access capabilities for the mobile and web clients.
-Developers can utilize it to build applications that will work independent of the network state.
-Showcase application contains integration with all offline capabilities offered by Data Sync Framework.
-In footer we should see current network state and number of offline changes we currently have.
+== Protect the mobile showcase app by Identity Management service
 
-image::images/offline-footer.png[integration, role="integr8ly-img-responsive"]
+The showcase app is working, but not secure. Let's use the Identity Management service to protect both the client and the data sync server.
 
-=== Steps
+=== Add Identity Management service to the client app
 
-. Let's open create task view and fill all task details
-. Please do not press create yet
-. To leverage offline capabilities we need to make sure that our application looses conectivity with our server. 
-On the Mobile phone this can be done by turning on airplane mode. 
-In Chrome browser you can use the Network tab from Chrome Developer Tools to simulate offline mode. (open them using F12, or using Command+Option+I on macOS)
-. Make your device go offline
-. Press `Create` button to create an new task
-. We should see task created and our `Offline Changes` button in footer containing now one change.
-. Let's edit task multiple times
-. Each edit will create new Task
-. We can review all offline items directly by clicking on `Offline Changes` button.
-. Please restore connectivity (depending on your device)
-. We should see all `Offline Changes` reflected back to server and appearing in second instance of the application.
-
-[time=6]
-== Conflict resolution functionalities
-
-Data Conflicts can happen when resource we modified was also modified on server.
-DataSync Framework enables multiple strategies to deal with conflicts. 
-By default Showcase will resolve conflicts by applying all server side changes on top of the client.
-In case of direct conflicts client field value will be used. 
-To simulate conflict we need to go offline and make edits in two instance of the application at the same time.
+. Go to the link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"].
+. Click on the *showcase-app* card to display the configuration screen.
+. Click the *Mobile Services* tab.
+. Click *Bind to App* for *Identity Management* in the *Unbound Services* section, with the following parameters:
+.. Admin User Name: `admin`
+.. Admin User Password: `admin`
+.. Client Type: `public`
+. Close the wizard. Once the binding process is completed, you should the *Identity Management* row is moved to the `Bound Services` section.
+. Go back to the `Configuration` tab of the app and you should see the content of `mobile-services.json` is updated. Copy the content and save it somewhere on the computer.
+
+=== Add Identity Management service to the data sync server
```

Comment:
This is actually for both client and server. We can split them or have them together (which makes sense)
```suggestion
=== Add Identity Management service to the showcase client and server applications
```